modules/installation-aws-permissions.adoc

```diff
@@ -266,18 +279,16 @@ If you use an existing VPC, your account does not require these permissions to d
 [%collapsible]
 ====
 * `iam:DeleteAccessKey`
```

Comment:
`iam:DeleteAccessKey` should be removed from `.Additional IAM and S3 permissions that are required to create manifests`